Wikipedia/Code/VanishAccountPopUpAlertView.swift

```diff
@@ -11,10 +11,6 @@ struct VanishAccountPopUpAlertView: View {
     
     enum LocalizedStrings {
         static let title = WMFLocalizedString("vanish-modal-title", value: "Vanishing request", comment: "Title text fot the vanish request modal")
-        static let firstItem = WMFLocalizedString("vanish-modal-item", value: "If you completed your vanishing request, please allow a couple of days for the request to be processed by an administrator.", comment: "Text indicating that the process of vanishing might take days to be completed")
-        static let secondItem = WMFLocalizedString("vanish-modal-item-2", value: "If you are unsure if your request went through please check your Mail app", comment: "Text indicating that the user should check if their email was sent in the Mail app used to send the message")
```

Comment:
@mazevedofs Are these needed?

Reply:
No, I moved them to the ListView struct and they were duplicated as well

Reply:
Nevermind! I didn't look close enough.